extensions/omni_ext/test/tests/omni_ext_test/tests.yml

```diff
@@ -42,9 +53,34 @@ tests:
     - query: select omni_ext_test.wait_for_table('local_worker_started')
       results:
       - wait_for_table: true
-  
+    - query: select n.nspname, c.relname from pg_catalog.pg_class c left join pg_catalog.pg_namespace  n on n.oid=c.relnamespace where c.relname='local_worker_started'
```

Comment:
@yrashk turned out we can't exactly re-implement describe (`\d`) with only SQL, I'm not sure what its role was in the pg regress tests, so I've only added these queries here to check existence of the table.

Reply:
Can you please elaborate on what exactly can't be implemented when it comes to `\d`?

Reply:
As for the role of those tests, do you mean this part?

```
\d local_worker_started;
      Table "public.local_worker_started"
 Column | Type | Collation | Nullable | Default 
--------+------+-----------+----------+---------
```

If yes, then yeah, we're just checking for the presence.

Reply:
> Can you please elaborate on what exactly can't be implemented when it comes to \d?

One immediate example was how \d supports describing other objects aside from tables (views, sequences, etc). Since we are only using it to check for presence though, I think the current query works and it's no longer a problem.

Reply:
Yeah, I think that use of `\d` was just a "lazy" version of `\dt`